frontend/openchat-client/src/openchat.ts

```diff
@@ -1430,15 +1433,15 @@ export class OpenChat extends EventTarget {
 
         setSelectedChat(this.api, clientChat, serverChat, messageIndex);
 
-        const { selectedChat, focusMessageIndex, events } = this._liveState;
+        const { selectedChat, focusMessageIndex } = this._liveState;
```

Comment:
The events here are always empty since we've just reset the selected chat, so we're safe to remove them.